tiledb/sm/array_schema/attribute.h

```diff
@@ -158,6 +165,12 @@ class Attribute {
    */
   bool var_size() const;
 
+  /**
+   * Returns *true* if this is a nullable attribute, and *false*
+   * otherwise.
+   */
+  bool nullable() const;
```

Comment:
We're not extra consistent here.  It is `set_fill_value()` and `get_fill_value()` yet also `filter_list()` (to set) and `get_filter_list()`.  

I seem to have a vague recollection that we decided that explicit is better.